OrbitQt/orbiteventiterator.ui

```diff
@@ -69,23 +69,19 @@
     </widget>
    </item>
    <item row="0" column="3">
-    <widget class="QLabel" name="position_label_">
```

Comment:
I somehow came to the conclusion that it's the default value. But I'm not so sure anymore. I will have a look again. Thanks.

Reply:
You were right. That fix was not correct, because the default is Qt::AlignLeft not Qt::AlignRight. To make the designer happy another flag "Qt::AlignTrailing" was needed. This flag means the same as Qt::AlignRight for LTR-writing systems.